src/components/grids.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const gapSize = 20
+const numberOfColumns = 12
+
+export const Row = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-right: -${gapSize / 2}px;
+  margin-left: -${gapSize / 2}px;
+  margin-bottom: ${gapSize}px;
+`
+
+export const Col = styled.div`
+  box-sizing: border-box;
+  flex: 0 0 auto;
+  padding-right: ${gapSize / 2}px;
+  padding-left: ${gapSize / 2}px;
+  flex-basis: ${props => (100 / numberOfColumns) * (props.size)}%;
+  max-width: ${props => (100 / numberOfColumns) * (props.size)}%;
```

Comment:
It seems like the value of `flex-grow` is relative to other columns while the current grid system has the fixed scale of 12 in total. We could also just use `flex` as suggested here: https://facebook.github.io/react-native/docs/height-and-width.html, but I'm not sure what it'd look like on modern browsers.

I feel like we should experiment with `flex` or other flex-related properties before we get too far so we can make grid perfectly re-usable across multiple platforms.

What are the browser versions we are targeting? Also, is there any example iOS app that actually uses reaction-force that we can use for acceptance?